pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -1043,37 +1050,53 @@ def adjoint(self):
         )
 
     # pylint: disable=unused-argument
-    def decomposition(self, *args, **kwargs):
+    def compute_decomposition(wires, **hyperparams):
+        """Compute the decomposition of the MultiControlledX gate.
+
+        Args:
+            wires: (Union[Wires or int]) The single target wire 
+        
+        Keyword Args:
+            control_wires 
+            work_wires
+            control_values
+            target_wire
```

Comment:
these are missing types and descriptions!

Reply:
And the return statement...

I was wondering, I thought our convention is to not distinguish args/kwargs in the docstirng?